docs/usage/user-stories/maintaining-aur-packages-with-renovate.md

```diff
@@ -0,0 +1,172 @@
+<!-- hide table of contents in navigation sidebar -->
+<style>
+.md-nav--primary .md-nav__link[for=__toc] ~ .md-nav {
+    display: none;
+}
+</style>
+
+# Maintaining AUR packages with Renovate
+
+> This article was written by [Jamie Magee](https://github.com/JamieMagee) and originally published on [Jamie Magee's blog](https://jamiemagee.co.uk/blog/maintaining-aur-packages-with-renovate/).
+
+<!-- prettier-ignore -->
+!!! note
+    Jamie Magee helps to maintain Renovate.
+    They obviously like Renovate, and want you to use it.
+
+One big advantage that Arch Linux has over other distributions, apart from being able to say “BTW I use arch”, is the Arch User Repository (AUR).
+It’s a community-driven repository with over 80,000 packages.
+If you’re looking for a package, chances are you'll find it in the AUR.
+
+Keeping all those packages up to date, takes a lot of manual effort by a lot of volunteers.
+People have created and used tools, like [`urlwatch`](https://github.com/thp/urlwatch) and [`aurpublish`](https://github.com/eli-schwartz/aurpublish), to let them know when upstream releases are cut and automate some parts of the process.
+I know I do.
+But I wanted to automate the entire process.
+I think [Renovate](https://github.com/renovatebot/renovate/) can help here.
+
+## Updating versions with Renovate
+
+Renovate is an automated dependency update tool.
+You might have seen it opening pull requests on GitHub and making updates for npm or other package managers, but it’s a lot more powerful than just that.
+
+Renovate has a couple of concepts that I need to explain first: [datasources](../modules/datasource/index.md) and [managers](../modules/manager/index.md).
+Datasources define where to look for new versions of a dependency.
+Renovate comes with over 50 different datasources, but the one that is important for AUR packages is the [`git-tags` datasource](https://docs.renovatebot.com/modules/datasource/#git-tags-datasource).
```

Comment:
The source article says "over 40 different datasources", but I'm counting 50+ datasources on the datasources homepage in the docs. Please check my counting. 😄

Reply:
I started writing this article a while ago. That probably accounts for the difference.